### ISSUE TITLE
renovation - PRJS_v1.0

### ISSUE THREAD

**Issue by ran-codes** [Drexel-UHC/salurbal-fair-renovations#58]
## Action Items 
  - [x] ETL setup
  - [x] define data-metadata relationships -> **linkage.csv**
  - [x] generate codebook templates -> **4.0-template__codebooks.xlsx**
  - [x] initial filling out of codebooks  -> **4.1-raw__codebooks.xlsx**
  - [x] QC of initial codebooks -> **4.2-base__codebooks.xlsx**
  - [x] Review of base codebooks  -> **4.3-int__codebooks.xlsx**
  - [ ] QC of intermediate codebooks  -> **4.4-final__codebooks.xlsx**
  - [ ] Integration and loading into data warehouse

**Comment by juruchi** [comment]
@goro-yamada The codebook is ready for your review. Details on codebook fields can be found [here](https://github.com/Drexel-UHC/salurbal-fair-renovations/raw/main/documents/templates/_template_details.xlsx) Let us know if you have any questions!

## Action items

- [ ] Review [4.2-base__codebook.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/blob/main/datasets/PRJS/v1.0/4.2-base__codebook.xlsx)
- [ ] Make edits as needed and save in a file called **4.3-int__codebook.xlsx**
- [ ] Comment in this issue when the final version is ready and upload the file to this issue via drag and drop

**Comment by goro-yamada** [comment]
@juruchi 
1. Codebook: 
a. I corrected the accents of the source institutions.
b. by_var_strata: I do not know how this will be shown in the codebook for the public, but note that the variables are sorted by their name that do not name age with leading zero. For this, variables for ages 2-9 are located strangely.
2. src__population.xlsx: I think I reviewed this for other population data. It looks good.

[4.3-int__codebook.xlsx](https://github.com/Drexel-UHC/salurbal-fair-renovations/files/11653822/4.3-int__codebook.xlsx)
